pkg/features/antrea_features.go

```diff
@@ -70,8 +70,30 @@ var (
 		FlowExporter:       {Default: false, PreRelease: featuregate.Alpha},
 		NetworkPolicyStats: {Default: false, PreRelease: featuregate.Alpha},
 	}
+
+	// UnsupportedFeaturesOnWindows records the features not supported on
+	// a Windows Node. Antrea Agent on a Windows Node checks the enabled
+	// features, and fails the startup if a unsupported feature is enabled.
+	// We do not define a separate defaultAntreaFeatureGates map for
+	// Windows, because Agent code assumes all features are registered (
+	// FeatureGate.Enabled(feature) will panic if the feature is not added
+	// to the FeatureGate).
+	// In future, if a feature is supported on both Linux and Windows, but
+	// can have different FeatureSpecs between Linux and Windows, we should
+	// still define a separate defaultAntreaFeatureGates map for Windows.
+	unsupportedFeaturesOnWindows = map[featuregate.Feature]bool{}
```

Comment:
Is it empty on purpose for now?
Using bool as the value may be confusing that the value indicates whether the feature is unsupported, maybe use struct{} like other set's case?

Reply:
Yes, until new features are added like IPv6, NodePortLocal, etc. we do not really have features unsupported on Windows.
Thought bool is simpler for adding new entries, but sure I can change to struct{}.

Reply:
It's also simple to add entries when value is struct{}.
```
unsupportedFeaturesOnWindows = map[featuregate.Feature]struct{}{
    XXXX: {},
    YYYY: {},
}
```

Reply:
Changed to struct{} already.